apps/docs/src/guide/cli/commands.md

```diff
@@ -50,13 +50,35 @@ In a nutshell:
 
 ## `fuels build`
 
+```console
+npx fuels help build
+```
+
+```
+Options:
+  -p, --path <path>  Path to project root (default: "/Users/anderson/Code/fuel/fuels-ts/apps/docs")
+  -d, --deploy       Deploy contracts after build (auto-starts a `fuel-core` node if needed)
```

Comment:
Just an observation:

It would be nice to have some way to use the text in 
```ts
.option('-d, --deploy', 'Deploy contracts after build (auto-starts a `fuel-core` node if needed)')
```
here as well, instead of copy-pasting.

Reply:
It would, not only here but in a lot of doc pages.

I don't know how that would be possible.

Probably not worth the trouble.

Reply:
Maybe the usage of regions, something like:
```ts
const fuelsBuildOptions: Record<string, { option: string; description: string }> = {
  path: {
    option:
      // #region fuels-build-options-path-option
      '-p, --path <path>',
    // #endregion
    // #region fuels-build-options-path-description
    description: 'Path to project root',
    // #endregion
  },
  deploy: {
    // #region fuels-build-options-deploy-option
    option: '-d, --deploy',
    // #endregion
    // #region fuels-build-options-deploy-description
    description: 'Deploy contracts after build (auto-starts a `fuel-core` node if needed)',
    // #endregion
  },
};
```